draft-ietf-dprive-dnsoquic.md

```diff
@@ -113,6 +113,13 @@ application will use QUIC. This is done for HTTP in "Hypertext Transfer
 Protocol Version 3 (HTTP/3)"{{?I-D.ietf-quic-http}}. The purpose of this
 document is to define the way DNS messages can be transmitted over QUIC.
 
+Future work might specify DNS over HTTP/3 for one or all of the scenarios
```

Comment:
DNS over H3, but also DoT of course. That begs the question of why do we say something about one versus the other...

Reply:
@huitema I think we already have several references to "QUIC vs TLS" throughout the Introduction and Design Considerations?

Reply:
Yeah, I don't think that we need to worry excessively about this one.  I'm just looking to avoid directly misleading statements.

> DNS over HTTP {{?DoH=RFC8484}} can be used with HTTP/3 to get some of the benefits of QUIC.  However, a lightweight [...]

Reply:
Thanks - change pushed. @huitema are you happy with this PR now?